moped-editor/src/views/projects/Toolbar.js

```diff
@@ -37,6 +37,7 @@ const Toolbar = ({ change, className, ...rest }) => {
                     variant="contained"
                     component={RouterLink}
                     to={"/moped/projects/new"}
+                    startIcon={<Icon>add_circle</Icon>}
```

Comment:
I was going to recommend importing like below but then I saw the `Icon` component in the MUI docs. TIL!

`import AddCircleIcon from "@material-ui/icons/AddCircle";`

`<AddCircleIcon color="primary" />`